plugin.rb

```diff
@@ -122,7 +122,7 @@ class Engine < ::Rails::Engine
   end
 
   add_to_serializer(:topic_list_item, :expert_post_group_names) do
-    object.custom_fields[CategoryExperts::TOPIC_EXPERT_POST_GROUP_NAMES]&.split("|")
```

Comment:
The JS tests didn't fail b/c we fake the server responses... This already gets stored as an array; doesn't need the splitting.